hypershift-operator/controllers/nodepool/nodepool_controller.go

```diff
@@ -1303,6 +1304,17 @@ func (r *NodePoolReconciler) getConfig(ctx context.Context,
 		configs = append(configs, *configConfigMap)
 	}
 
+	// Look for operator generated MachineConfigs from the hosted control plane namespace
+	operatorGeneratedConfigs := &corev1.ConfigMapList{}
```

Comment:
`operatorGeneratedConfigs`, which operator?

Reply:
Thanks, as discussed on slack I was originally thinking it may be useful to keep this generic, but since we currently have no other need for this functionality, I renamed this variable to be NTO specific (`nodeTuningGeneratedConfigs`)